docs/data/stashcache/overview.md

```diff
@@ -0,0 +1,16 @@
+# Overview of StashCache
+
+StashCache is a data caching federation with many distributed caches spread across the U.S.
+
+
+![StashCache Map](StashCacheMap.png "StashCache Map")
+
+The StashCache federation consists of two service types, the Caches and the Origins:
+
+* StashCache Cache: role of **"cache"** server is to keep data cached and immediately available (via [stashcp](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcach) or CVMFS) within Stash federation (without re-transferring from "origin").
```

Comment:
Presumably not all admins should install caches. What conditions would prompt an admin to set up a cache?

Reply:
Not all should, probably.  But any can.  Changed.